docusaurus/docs/dev-docs/api/plugins/admin-panel-api.md

```diff
@@ -233,18 +193,14 @@ The Settings API allows:
 * [creating a new setting section](#createsettingsection)
 * adding [a single link](#addsettingslink) or [multiple links at once](#addsettingslinks) to existing settings sections
 
-:::note
-Adding a new section happens in the [register](#register) lifecycle while adding links happens during the [bootstrap](#bootstrap) lifecycle.
-:::
-
-All functions accept links as objects with the following parameters:
+All functions expect an object with the following attributes:
```

Comment:
```suggestion
:::info 
Adding a new section happens in the [register](#register) lifecycle while adding links happens during the [bootstrap](#bootstrap) lifecycle.
::: 

All functions expect an object with the following attributes:
```

(to be in line with what was discussed [here](https://github.com/strapi/documentation/pull/1822/files#r1300135370) ^^)